core/src/test/java/org/mskcc/cbio/portal/util/TestGenePanelUtil.java

```diff
@@ -0,0 +1,242 @@
+package org.mskcc.cbio.portal.util;
+
+import org.junit.Test;
+import org.mskcc.cbio.portal.model.CanonicalGene;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mskcc.cbio.portal.util.GenePanelUtil.*;
+
+public class TestGenePanelUtil {
+
+    CanonicalGene otor = makeGene(13321, 56914, "OTOR", makeAliases("FDP", "MIAL1"));
+    CanonicalGene cadm2 = makeGene(20001, 253559, "CADM2", makeAliases("IGSF4D", "Necl-3", "NECL3", "SynCAM 2", "synCAM2"));
+    CanonicalGene msh3 = makeGene(3573, 4437, "MSH3", makeAliases("MRP1", "DUP", "FAP4"));
+    CanonicalGene p2ry10 = makeGene(10455, 27334, "P2RY10", makeAliases("P2Y10", "LYPSR2"));
+    CanonicalGene adamts20 = makeGene(15009, 80070, "ADAMTS20", makeAliases("ADAM-TS20", "GON-1", "ADAMTS-20"));
+    CanonicalGene kat2a = makeGene(2154, 2648, "KAT2A", makeAliases("hGCN5", "GCN5", "GCN5L2", "PCAF-b"));
+    CanonicalGene myb = makeGene(3674, 4602, "MYB", makeAliases("c-myb_CDS", "Cmyb", "c-myb", "efg"));
+    CanonicalGene npipb15 = makeGene(24049, 440348, "NPIPB15", makeAliases("A-761H5.4", "NPIPL2"));
+    CanonicalGene dtnb = makeGene(1492, 1838, "DTNB", null);
+    CanonicalGene ablim1 = makeGene(3239, 3983, "ABLIM1", makeAliases("LIMAB1", "abLIM-1", "ABLIM", "LIMATIN"));
+    CanonicalGene piezo1 = makeGene(7609, 9780, "PIEZO1", makeAliases("FAM38A", "Mib", "LMPH3", "DHS"));
+    CanonicalGene fgfr3 = makeGene(1827, 2261, "FGFR3", makeAliases("JTK4", "HSFGFR3EX", "ACH", "CEK2", "CD333"));
+
+    public TestGenePanelUtil() {
```

Comment:
Delete

Reply:
Do you mean the class constructor or the genes? The genes are used in the tests but deleting the constructor still lets the tests run and pass